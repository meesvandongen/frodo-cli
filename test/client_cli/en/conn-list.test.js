import cp from 'child_process';
import { promisify } from 'util';

const exec = promisify(cp.exec);
const CMD = 'frodo conn list --help';
const { stdout } = await exec(CMD);

test("CLI help interface for 'conn list' should be expected english", async () => {
  expect(stdout).toMatchSnapshot();
});
