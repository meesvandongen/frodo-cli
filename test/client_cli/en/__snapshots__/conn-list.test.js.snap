// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CLI help interface for 'conn list' should be expected english 1`] = `
"Usage: frodo conn list [options]

List connection profiles.

Options:
  --debug               Debug output during command execution. If specified,
                        may or may not produce additional output helpful for
                        troubleshooting.
  -h, --help            Help
  -l, --long            Long with all fields. (default: false)
  --sa-id <sa-id>       Service account id.
  --sa-jwk-file <file>  File containing the java web key (jwk) associated with
                        the the service account.
  --verbose             Verbose output during command execution. If specified,
                        may or may not produce additional output.
"
`;
